### PR TITLE
[fix] mlflow conversion with experiment id/name

### PR DESCRIPTION
**Pull request by devfox-se**

### CODE REVIEW

**Comment by devfox-se** [comment]
This resolves #1415

File: aim/cli/convert/processors/mlflow.py [hunk 1]
```diff
@@ -41,11 +41,17 @@ def parse_mlflow_logs(repo_inst, tracking_uri, experiment):
...
 
+    experiment = kwargs.get('experiment_id') or kwargs.get('experiment_name')
+    get_experiment_by_name = kwargs.get('experiment_name') == experiment
+
     if experiment is None:
         # process all experiments
         experiments = client.list_experiments()
     else:
-        experiments = (client.get_experiment_by_name(experiment),)
+        if get_experiment_by_name:
```
Comment (alberttorosyan) [inline aim/cli/convert/processors/mlflow.py:51]: lets use the single `--experiment` option.
try with name first, if experiment is not found try treat it as experiment ID
if the experiment is not found, echo an error message and exit

**Review by alberttorosyan — CHANGES_REQUESTED** [review]

File: aim/cli/convert/processors/mlflow.py (hunk 1 above)
Comment (devfox-se) [reply aim/cli/convert/processors/mlflow.py:51]: well yeah good idea

**Review by alberttorosyan — APPROVED** [review]
please update description for `--experiment` argument for CLI ref. and guides.

**Pull request merged**